CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+and one output. This may not always be possible and Bob may have to give up on being Alice's
+counter-party. In other words, because Alice was forced to specify placeholders in her intent, she
+missed out on a potential trading partner.
+
+DApps would also be unable to be direct liquidity sources for these intents due to not always being
+able to match the value specified in each placeholder. If the input place holder is for 10.3 ADA but
+the liquidity source UTxO actually has 20.5 ADA, this liquidity source cannot be used despite it
+having enough liquidity. The Validation Zones CIP argues that DApps can just add support for
+breaking-off/merging UTxOs so that the liquidity source UTxOs can be forced into the proper forms
+for the required placeholders. However, this would dramatically increase the complexity and attack
+surface of DApps that do this. And this added risk to users is just to compensate for a limitation
+in the Validation Zones' design.
+
+In addition to the counter-party issue, there must be a way for the ledger to actually support and
+satisfy these placeholders. To accomplish this, the Validation Zones CIP introduces zones, unlocked
+requests, and unlocked outputs. These new features require adding a lot of extra complexity to the
+ledger and makes using Validation Zones intents a very complicated experience. For example,
+satisfying a single intent actually requires multiple transactions (packaged as a zone).
+
+Again, the counter-party issue and extra complexity in Validation Zones are simply due to not being
+able to accurately model unbalanced intents. Since this CIP *can* model unbalanced intents, it can
+cover all of the same use cases while only adding a fraction of the complexity.
+
+It is also worth noting that, since the Validation Zones design uses otherwise unnecessary
+placeholders and extra transactions just to satisfy these placeholders, the impact Validation Zones
+would have on the average block size is likely equivalent to, if not larger than, the impact this
+CIP would introduce.
+
+### Westberg's "Smart Transactions"
+
+[Smart
+Transactions](https://github.com/input-output-hk/Developer-Experience-working-group/issues/47) is
+very similar to Validation Zones in that it tries to use fully balanced transactions to model all
+intents. Because of this, it suffers from the same counter-party issue, over-complexity, and
+possible overly-increased average block size as Validation Zones.
+
+### Original Babel Fees Design
+
+The original design effectively required minting policies to be executed in every transaction, but
+this is not always possible/feasible.
+
+## Path To Active
+
+### Acceptance Criteria
+
+- [] It is approved by the community and the ledger/plutus teams.
+
+### Implementation Plan
+
+- [] The ledger spec is updated to use transaction pieces.
+- [] Implementation of the Cardano node with a new era/hardfork.
+- [] Deployed on testnet/mainnet
```

Comment:
```suggestion
- [ ] It is approved by the community and the ledger/plutus teams.

### Implementation Plan

- [ ] The Ledger spec is updated to use transaction pieces.
- [ ] Implementation in the Cardano node with a new Ledger era / hardfork
- [ ] Deployed on testnets / Mainnet
```
Fixing typography & some spelling.  I do think we need more detail in _Acceptance Criteria_ but I think these will emerge from discussion points over a long Ledger + community review.